Modules/xASL_module_Structural.m

```diff
@@ -42,7 +42,15 @@
 result = false;
 
 [x] = xASL_init_SubStructs(x);
-x = xASL_init_InitializeMutex(x, 'T1'); % starts mutex locking process to ensure that everything will run only once
+[x, bLocked] = xASL_init_InitializeMutex(x, 'T1'); % starts mutex locking process to ensure that everything will run only once
```

Comment:
please remove bLocked.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test in detail.